yt_dlp/extractor/nuum.py

```diff
@@ -44,8 +44,11 @@ def _parse_video_data(self, container, extract_formats=True):
 
         formats, subtitles = None, None
         if extract_formats:
+            headers = {'Referer': 'https://nuum.ru/'}
             formats, subtitles = self._extract_m3u8_formats_and_subtitles(
-                media_url, video_id, 'mp4', live=is_live)
+                media_url, video_id, 'mp4', live=is_live, headers=headers)
+            for format in formats:
+                format.update({'http_headers': headers})
```

Comment:
IMO just return `'http_headers': {'Referer': 'https://nuum.ru/'},` in the info dict root instead